play/src/pages/gameinprogress/ChooseAnswer.tsx

```diff
@@ -69,6 +81,34 @@ export default function ChooseAnswer({
     </>
   );
 
+  const onSubmitDisplay =
+    currentState === GameSessionState.CHOOSE_CORRECT_ANSWER ? (
```

Comment:
Has the design team confirmed that this component can only appear in phase one of the game?

Reply:
Yep, Yong Lin specified the card should only display on phase 1

Reply:
Perfect, thanks for confirming